aip/general/0148.md

```diff
@@ -37,8 +37,10 @@ collection, and **should** be used in most `List` (AIP-132) and `Create`
 #### uid
 
 The output only `string uid` field refers to a system-assigned, unique
-identifier for a resource. When provided, this field **should** be a [UUID4][].
-[Declarative-friendly resources][] **should** include this field.
+identifier for a resource. When provided, this field **should** be a [UUID4][]
```

Comment:
Does this really need to be **should** or could we make it **must**?

Reply:
I have no good reason why it shouldn't be annotated as such. I've changed the guidance to use the format extension to a "must". Done!